cmd/template/echoRoutes.go

```diff
@@ -0,0 +1,43 @@
+package template
+
+type EchoTemplates struct{}
+
+func (e EchoTemplates) Main() []byte {
+	return MainTemplate()
+}
+func (e EchoTemplates) Server() []byte {
+	return MakeHTTPServer()
+}
+
+func (e EchoTemplates) Routes() []byte {
+	return MakeEchoRoutes()
+}
+
+func MakeEchoRoutes() []byte {
+	return []byte(`package server
+
+	import (
+		"net/http"
+	
+		"github.com/labstack/echo/v4"
+		"github.com/labstack/echo/v4/middleware"
+	)
+func (s *Server) RegisterRoutes() http.Handler {
+	e := echo.New()
+	e.Use(middleware.Logger())
+	e.Use(middleware.Recover())
```

Comment:
You can just call `echo.Default()` as it uses Logger and Recovery by default :P

Reply:
@xsadia Ah this is a good point
If we use echo. Defauflt() it comes with the logger from middleware 
so we potentially dont even need to include it as a second go get dep

Reply:
I may or not have been malding while looking at this and confused myself with Gin..